core/Inst.hpp

```diff
@@ -32,6 +32,32 @@ namespace olympia
     class Inst {
     public:
 
+        class RenameData{
+            public:
+            void setDestination(unsigned int destination){
+                dest = destination;
+            }
+            void setOriginalDestination(unsigned int destination){
+                original_dest = destination;
+            }
+            void setSource(std::pair<unsigned int, bool> source){
+                src.push_back(source);
+            }
+            std::vector<std::pair<unsigned int, bool>> getSource(){
+                return src;
+            }
+            unsigned int getDestination(){
+                return dest;
+            }
+            unsigned int getOriginalDestination(){
+                return original_dest;
+            }
+            private:
+            unsigned int dest;
+            unsigned int original_dest;
+            std::vector<std::pair<unsigned int, bool>> src;
```

Comment:
Private methods and variables end with an `_`.

Reply:
Addressed in [163a70b](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/163a70b5862658deca0df16b035bbe59dc841c92)